plugins/@grouparoo/pipedrive/src/lib/common/listMethods.ts

```diff
@@ -1,53 +1,36 @@
-import { objectCache } from "@grouparoo/core";
 import { PipedriveClient } from "../client";
 import {
-  getKnownPersonFieldMap,
+  getKnownFieldMap,
   PipedriveCacheData,
 } from "./destinationMappingOptions";
+import { configMapping, EntityType } from "./configMapping";
+import { objectCache } from "@grouparoo/core";
 
-export async function getGroupFieldKey(
-  client: PipedriveClient,
-  cacheData: PipedriveCacheData,
-  groupName: string,
-  createIfNotExists?: boolean
-): Promise<string> {
-  groupName = groupName.trim();
+export const GROUP_FIELD_PREFIX = "In Group:";
 
-  const { appId, appOptions } = cacheData;
-  const cacheDurationMs = 1000 * 60 * 10; // 10 minutes
-  const cacheKey = ["getGroupFieldKey", groupName, appOptions];
-  const write = createIfNotExists; // don't cache nulls
-  return objectCache(
-    { objectId: appId, cacheKey, cacheDurationMs, write },
-    async () => {
-      return ensureFieldAndFilter(
-        client,
-        cacheData,
-        groupName,
-        createIfNotExists
-      );
-    }
-  );
+export function makeGroupFieldName(groupName: string) {
+  return `${GROUP_FIELD_PREFIX} ${groupName}`;
 }
 
 // gets called if the group field key is not cached
-async function ensureFieldAndFilter(
+export async function ensureFieldAndFilter(
```

Comment:
is this used in another file?

Reply:
Nope, I've checked it and all other exported ones.